src/+otp/+zlakinetics/ZLAKineticsProblem.m

```diff
@@ -0,0 +1,46 @@
+classdef ZLAKineticsProblem < otp.Problem
+  %
+  % Problem description and fortran implementation  can be found in
+  % Francesca Mazzia, Cecilia Magherini and Felice Iavernaro
+  % Test Set for Initial Value Problem Solvers, 2006
+  % https://archimede.dm.uniba.it/~testset/report/chemakzo.pdf
+
+    methods
+        function obj = ZLAKineticsProblem(timeSpan, y0, parameters)
+            obj@otp.Problem('ZLA Kinetics', 6, timeSpan, y0, parameters);
+        end
+    end
+    
+    methods (Access = protected)
+        function onSettingsChanged(obj)
+            k = obj.Parameters.k;
+            K = obj.Parameters.K;
+            KlA = obj.Parameters.KlA;
+            pCO2 = obj.Parameters.pCO2;
+            H = obj.Parameters.H;
+            Ks = obj.Parameters.Ks;
+
+            
+            obj.Rhs = otp.Rhs(@(t, y) otp.zlakinetics.f(t, y, k, K, KlA, pCO2, H, Ks), ...
+                otp.Rhs.FieldNames.Jacobian, @(t, y) otp.zlakinetics.jac(t, y, k, K, KlA, [], [], Ks), ...
```

Comment:
instead of passing [], I think as long as we have "~" in the method it would serve the same function, and not actually pass the value, but would look cleaner here.